docs/user/production-considerations/task-manager-production-considerations.asciidoc

```diff
@@ -2,7 +2,7 @@
 [[task-manager-production-considerations]]
 == Task Manager
 
-{kib} Task Manager is leveraged by features such as Alerting, Actions, and Reporting to run mission critical work as persistent background tasks.
+{kib} Task Manager is leveraged by features such as Alerting, Actions, Reporting, and Telemetry to run mission critical work as persistent background tasks.
```

Comment:
Why is `Telemetry` included in this list of "mission critical work" and not in the other lists of "mission critical services"?

Reply:
Good catch, I don't think I read the rest of the sentence 🙈 Fixed in https://github.com/elastic/kibana/pull/130317/commits/f8d0e9bd27a48b51041ff500541dc20c21f14cf9.